erpnext/hr/doctype/hr_settings/hr_settings.json

```diff
@@ -131,25 +130,13 @@
    "fieldtype": "Link",
    "label": "Role Allowed to Create Backdated Leave Application",
    "options": "Role"
-  },
-  {
-   "default": "0",
-   "fieldname": "automatically_allocate_leaves_based_on_leave_policy",
-   "fieldtype": "Check",
-   "label": "Automatically Allocate Leaves Based On Leave Policy"
-  },
-  {
-   "default": "1",
-   "fieldname": "send_leave_notification",
```

Comment:
I think the **Send Leave Notification** configuration got removed by mistake.

Reply:
added again